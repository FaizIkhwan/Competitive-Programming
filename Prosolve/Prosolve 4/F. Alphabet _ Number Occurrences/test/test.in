2
Hello Python 3333
Abc123
