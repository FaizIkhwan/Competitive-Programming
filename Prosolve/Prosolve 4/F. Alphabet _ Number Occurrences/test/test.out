Alphabet: 11, Digit: 4
Alphabet: 3, Digit: 3
